tests/TestUtility/OmniSharpTestHost.cs

```diff
@@ -136,7 +136,12 @@ public void AddFilesToWorkspace(params TestFile[] testFiles)
                 this.Workspace,
                 "project.json",
                 new[] { "dnx451", "dnxcore50" },
-                testFiles);
+                testFiles.Where(f => f.FileName.EndsWith(".cs", StringComparison.CurrentCultureIgnoreCase)).ToArray());
```

Comment:
ah sure, I just picked "that long thing" from intellisense without thinking twice